model/application.go

```diff
@@ -13,8 +13,8 @@ type Application struct {
 	Facts             JSON
 	Comments          string
 	Tasks             []Task     `gorm:"constraint:OnDelete:CASCADE"`
-	Tags              []Tag      `gorm:"many2many:applicationTags;constraint:OnDelete:CASCADE"`
-	Identities        []Identity `gorm:"many2many:appIdentity;constraint:OnDelete:CASCADE"`
+	Tags              []Tag      `gorm:"many2many:ApplicationTags;constraint:OnDelete:CASCADE"`
+	Identities        []Identity `gorm:"many2many:ApplicationIdentity;constraint:OnDelete:CASCADE"`
```

Comment:
The created association table should be upper cased for consistency with other tables.